flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/pom.xml

```diff
@@ -99,6 +107,19 @@ limitations under the License.
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-connector-files</artifactId>
+            <version>${flink.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this used somewhere?

Reply:
Yeah, if removed, will got the following exception:
![image](https://github.com/user-attachments/assets/b774d082-b903-440e-a5a0-74c11084fd5e)

And flink-connector-files is also provided in FLINK_HOME/lib directory.

Reply:
I see, `paimon-flink-common` declares it as a provided dependency.